docs/CH-Edgex-taf.rst

```diff
@@ -0,0 +1,152 @@
+#########
+EdgeX-taf
+#########
+
+Requirements:
+
+* Well-defined project structure
+* Test cases written using human-readable syntax (robotframework does this)
+* Run test scripts using robotframework or dockerized robotframework
+
+  - Deploy test environment ( edgex services for testing)
+  - Run test
+  - Shutdown test environment
+
+* Generate report (robotframework does this)
+* All components are dockerized
+* Integration with Jenkins
```

Comment:
We need a separate PR for the integration to Jenkins.

Reply:
I create an issue for this task https://github.com/edgexfoundry/edgex-taf/issues/11.